partner_email_duplicate_warn/views/res_partner.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_partner_form" model="ir.ui.view">
```

Comment:
```suggestion
    <record id="view_partner_form_with_email_warning" model="ir.ui.view">
```

This view was renamed in v17. 
Because the view does not match, and same_email_partner_id was renamed in same_email_partner_ids, I get an error during the upgrade.